src/args.rs

```diff
@@ -214,6 +214,17 @@ pub struct CliArgs {
     #[arg(short = 'z', long = "enable-zip", env = "MINISERVE_ENABLE_ZIP")]
     pub enable_zip: bool,
 
+    /// Enable compress response
+    ///
+    /// WARNING: Enabling this option may slow down transfers due to CPU overhead, so it is disabled by default.
+    ///
+    /// Only enable this option if you know that your users have slow connections or if you want to minimize your server's bandwidth usage.
+    #[arg(
+        long = "enable-compress-response",
+        env = "MINISERVE_ENABLE_COMPRESS_RESPONSE"
+    )]
+    pub enable_compress_response: bool,
+
```

Comment:
What do you think about just calling this `--compress-response`? The `--enable` prefixes are only there for archiving. Also, let's also give it `short = 'C'`.

Reply:
I think that's a great idea. I've actually considered doing something like that before.